addons/mail/static/src/components/activity/activity.scss

```diff
@@ -172,3 +172,12 @@ a:not([href]):not([tabindex]).o_Activity_detailsButton {
     color: gray('500');
 }
 
+.o_Activity_note, .o_mail_note_title + div {
+  p {
+    margin-bottom: 0;
+  }
+}
+
+.o_mail_note_title {
+  margin-top: map-get($spacers, 2);
+}
```

Comment:
Fix is nice. Would just re-organize a bit styles like this:

```scss
// ------
// Layout
// -------

// [AKU] put this style in alphabetic order
.o_Activity_note p {
   margin-bottom: 0;
}

// [AKU] put these styles at the end of category
// From python template
.o_mail_note_title {
   margin-top: map-get($spacers, 2);
}

.o_mail_note_title + div p {
   margin-bottom: 0;
}
```

_(without the [AKU] comments)_

Best to have simple and flat selectors, and the ones that apply on stuffs from python template is best to isolate from the rest (hence why I suggest putting it at the end).

I feel it's off to have this separation of component template and python templates... But this fix is acceptable considering it doesn't break old activity notes.

Reply:
I'll make the change myself, just comment fyi ;)